src/data/list/basic.lean

```diff
@@ -2924,6 +2924,14 @@ begin
         or_and_distrib_right, exists_or_distrib, this, exists_eq_left] }
 end
 
+@[simp] theorem filter_map_join (f : α → option β) (L : list (list α)) :
```

Comment:
This might be more suited to data/list/join

Reply:
Then we should probably move `map_join` to data/list/join as well.

Reply:
I'd like to hear opinion(s) about the move.

Reply:
The fact that `map_join` is in this file is convincing enough for me